auth/auth_test.go

```diff
@@ -697,25 +698,88 @@ func TestVerifyIDTokenWithNoProjectID(t *testing.T) {
 	}
 }
 
-func TestVerifyIDTokenInEmulatorMode(t *testing.T) {
-	os.Setenv(emulatorHostEnvVar, "localhost:9099")
-	defer os.Unsetenv(emulatorHostEnvVar)
+func TestVerifyIDTokenUnsigned(t *testing.T) {
```

Comment:
Also added a test to guarantee that the non-emulator mode implementation doesn't allow unsigned tokens.